sources/HUBViewModelBuilderImplementation.m

```diff
@@ -266,6 +266,22 @@ - (void)removeAllComponentModelBuilders
                                                        customData:[self.customData copy]];
 }
 
+#pragma mark - Manipulate custom data
+
+- (void)setCustomDataValue:(nullable id)value forKey:(nonnull NSString *)key
+{
+    NSMutableDictionary *customData = self.customData.mutableCopy;
```

Comment:
Since mutable copy is an action being taken on the dictionary, call it as a method with `[]` notation.